enzyme/Enzyme/EnzymeLogic.cpp

```diff
@@ -2407,68 +2407,68 @@ void createTerminator(TypeResults &TR, DiffeGradientUtils *gutils,
   IRBuilder<> nBuilder(nBB);
   nBuilder.setFastMathFlags(getFast());
 
```

Comment:
Add a comment that in forward mode only need to update the return

Reply:
done

Reply:
Mind removing "so we might return early", otherwise lgtm